codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/endpoint/EndpointsDecorator.kt

```diff
@@ -102,7 +102,6 @@ class EndpointsDecorator : ClientCodegenDecorator {
     override val name: String = "Endpoints"
     override val order: Byte = 0
 
-    // TODO(enableNewSmithyRuntime): Remove `operationCustomizations` and `InjectEndpointInMakeOperation`
```

Comment:
Things have changed since that comment was added. Previously, I thought we would remove `operationCustomizations`, but then I ended up adding orchestrator-specific operation customizations. So now, we'll just remove the middleware-specific customizations from it.